### PR TITLE
[DataExplorer] automatically load a file given as a command line argument

### PR DESCRIPTION
**Pull request by norihiro-w**
as titled. this is my first try and any suggestions are welcome.

### CODE REVIEW

File: Applications/DataExplorer/mainwindow.cpp [hunk 1]
```diff
@@ -1096,6 +1096,16 @@ void MainWindow::HideWindow()
 	this->hide();
 }
 
+void MainWindow::LoadFile(const QString &fileName)
+{
+	ImportFileType::type t;
+	QString ext = QFileInfo(fileName).suffix();
+	if (ext=="msh" || ext=="vtu" || ext=="gli" || ext=="gml") {
+		t = ImportFileType::OGS_MSH;
```
Comment (norihiro-w) [inline Applications/DataExplorer/mainwindow.cpp]: OGS_MSH -> OGS

**Comment by TomFischer** [comment]
Looks good. :+1:

**Comment by norihiro-w** [comment]
@TomFischer corrected

**Comment by norihiro-w** [comment]
rebased

File: Applications/DataExplorer/mainwindow.cpp [hunk 2]
```diff
@@ -1096,6 +1096,16 @@ void MainWindow::HideWindow()
 	this->hide();
 }
 
+void MainWindow::LoadFile(const QString &fileName)
```
Comment (rinkk) [inline Applications/DataExplorer/mainwindow.cpp]: there is already a ```loadFile()```-method and having another one (with capital L) is confusing.
maybe rename this to ```loadFileOnStartUp()``` or something ...

File: Applications/DataExplorer/mainwindow.cpp [hunk 3]
```diff
@@ -1096,6 +1096,16 @@ void MainWindow::HideWindow()
...
+{
+	ImportFileType::type t;
+	QString ext = QFileInfo(fileName).suffix();
+	if (ext=="msh" || ext=="vtu" || ext=="gli" || ext=="gml") {
+		t = ImportFileType::OGS;
+		this->loadFile(t,fileName);
```
Comment (rinkk) [inline Applications/DataExplorer/mainwindow.cpp]: you can completely remove the ```t``` variable and just write
```this->loadFile(ImportFileType::OGS, fileName);``` 
this saves 2 lines of code .... 3 if you also remove the braces ...

**Comment by norihiro-w** [comment]
updated according to the comments

**Pull request merged**